### ISSUE TITLE
Bad permissions reports on install sysdig debian package (0.1.99) on ubuntu 14.10

### ISSUE THREAD

**Issue by demarant** [draios/sysdig#347]
On Ubuntu 14.10 I downloaded
https://s3.amazonaws.com/download.draios.com/stable/deb/stable-amd64/sysdig-0.1.99-x86_64.deb
and tried to install it.
but I get "The package is of bad quality":
"The installation of a package which violates the quality standards isn't allowed. This could cause serious problems on your computer. Please contact the person or organisation who provided this package file and include the details beneath."
and the following output:
Lintian check results for /home/marinis/sysdig-0.1.99-x86_64.deb:
E: sysdig: control-file-has-bad-permissions postinst 0644 != 0755
E: sysdig: control-file-has-bad-permissions prerm 0644 != 0755

So I did not install it.
What is the solution?

**Comment by gianlucaborello** [comment]
Do you have a way to replicate this? I can easily fix it but it doesn't seem to be a fatal issue for me, I just installed the same package on 14.10 and it goes through just fine:

```
$ sudo dpkg -i sysdig_0.1.99_amd64.deb
Selecting previously unselected package sysdig.
(Reading database ... 56701 files and directories currently installed.)
Preparing to unpack .../sysdig_0.1.99_amd64.deb ...
Unpacking sysdig (0.1.99) ...
Setting up sysdig (0.1.99) ...
Loading new sysdig-0.1.99 DKMS files...
First Installation: checking all kernels...
Building only for 3.16.0-30-generic
Building initial module for 3.16.0-30-generic
Done.

sysdig-probe:
Running module version sanity check.
 - Original module
   - No original module exists within this kernel
 - Installation
   - Installing to /lib/modules/3.16.0-30-generic/updates/dkms/

depmod....

DKMS: install completed.
Processing triggers for man-db (2.7.0.2-2) ...
```

For clarity, I can correctly see all the lintian rant, if I look for it:

```
$ lintian sysdig_0.1.99_amd64.deb
E: sysdig: embedded-library usr/bin/sysdig: libjsoncpp
E: sysdig: embedded-library usr/bin/sysdig: zlib
W: sysdig: hardening-no-relro usr/bin/sysdig
E: sysdig: missing-dependency-on-libc needed by usr/bin/sysdig
E: sysdig: changelog-file-missing-in-native-package
E: sysdig: file-in-etc-not-marked-as-conffile etc/bash_completion.d/sysdig
E: sysdig: control-file-has-bad-permissions postinst 0644 != 0755
E: sysdig: control-file-has-bad-permissions prerm 0644 != 0755
E: sysdig: no-copyright-file
E: sysdig: misplaced-extra-member-in-deb _gpgbuilder (unexpected member at position 3)
E: sysdig: description-starts-with-package-name
E: sysdig: extended-description-is-empty
W: sysdig: extra-license-file usr/share/sysdig/chisels/COPYING
E: sysdig: manpage-not-compressed usr/share/man/man8/sysdig.8
W: sysdig: manpage-has-errors-from-man usr/share/man/man8/sysdig.8 28: warning [p 1, 4.2i]: can't break line
W: sysdig: binary-without-manpage usr/bin/sysdig-probe-loader
```

**Comment by demarant** [comment]
@gianlucaborello:
Yes, it can be replicated.
Instead of using the shell command dpkg use the standard Ubuntu GUI Software Center, for example when you download from the browser and just click on the .deb package, the software center will start and you choose "install". There Ubuntu software center will display the "nasty warning dialog message" with title "the package is of bad quality..."
It is sad to get such a message when it is just about fixing some permissions on filea., so I thought it is worth reporting :)

**Comment by gianlucaborello** [comment]
That specific warning should be fixed with 48955f56e7077304047578d439613eec4fc004f3, thanks for filing the issue and please report if the other warnings are showing up as well, since it's indeed nasty and we'll try to fix as many as possible if they are shown to the user.
